msfs_derive/src/lib.rs

```diff
@@ -41,18 +41,14 @@ pub fn standalone_module(_args: TokenStream, item: TokenStream) -> TokenStream {
             },
         };
 
-        #[no_mangle]
-        pub extern "C" fn module_init() {
-            unsafe {
-                ::msfs::wrap_executor(&raw mut #executor_name, |e| e.handle_init());
-            }
+        #[unsafe(no_mangle)]
+        pub unsafe extern "C" fn module_init() {
+            ::msfs::wrap_executor(&raw mut #executor_name, |e| e.handle_init());
         }
 
-        #[no_mangle]
-        pub extern "C" fn module_deinit() {
-            unsafe {
-                ::msfs::wrap_executor(&raw mut #executor_name, |e| e.handle_deinit());
-            }
+        #[unsafe(no_mangle)]
+        pub unsafe extern "C" fn module_deinit() {
```

Comment:
```suggestion
        pub extern "C" fn module_deinit() {
```
The `unsafe` is not necessary.